app/concerns/orderable.rb

```diff
@@ -20,12 +20,21 @@ def reorder_others(is_after)
       other_where = Hash[ordered_attributes.map do |atr|
         [atr, send(is_after ? "#{atr}_before_last_save" : "#{atr}_was")]
       end]
-      others = self.class.where(other_where).order('section_order asc')
+      if self.is_a?(Reply)
+        others = self.class.where(other_where).ordered
+      else
+        others = self.class.where(other_where).order('section_order asc')
+      end
       return unless others.present?
 
       others.each_with_index do |other, index|
-        next if other.section_order == index
-        other.section_order = index
+        if self.is_a?(Reply)
```

Comment:
Also not Good Style. We could implement a section_order= method on Reply that set reply_order, or define a class method to define this, or do the same default-overriden-in-Reply pattern above, or even just rename this field to 'order' everywhere or something. Or pass the name of the method to the concern? Many options.

Reply:
I'm not sure what "do the same default-overriden-in-Reply pattern above" means?

I think defining `section_order` and `section_order=` methods on Reply would be the easiest route, but we might (in the long run) want to go about renaming the fields to `order`.